Ownership/AC - West/35.txt

```diff
@@ -7,4 +7,4 @@ SECTOR:London S35 (335-660):33500:66000
 OWNER:L35:LW:LWN:LWC:LWS:L
 ALTOWNER:Observing LON_W_CTR:L35:O:LM:LW:LWN:LWC:LWS:L
 ALTOWNER:Observing London FIR:O:LM:L35:LW:LWN:LWC:LWS:L
-BORDER:LWN1:LWN2:LWN3:LWC1:LWC2:LWC3a:LWC3b:LWC4a:LWC4b:LWC4c:LWC4d:LTC22c:LTC21a:LTC27:LWS2:LW8:LW9:LW4:BAN3:LW14:LW13
\ No newline at end of file
+BORDER:LWN1:LWN2:LWN3:LWC1:LWC2a:LWC2b:LWC3a:LWC3b:LWC4a:LWC4b:LWC4c:LW8:LW9:LW4:BAN3:LW14:LW13
```

Comment:
LWC2a is not defined in Lines - LACC.txt

O [20:22:44] Error: Invalid line name in sector BORDER: London S35 (335-660)/LWC2a in line 12432